.github/workflows/release.yml

```
@@ -134,9 +138,13 @@ jobs:
         with:
           path: ${{ env.IMGTOOL_PACKING_PATH }}
 
+      - name: Set the version
+        working-directory: ${{ env.IMGTOOL_PACKING_PATH }}/patches/
+        run: sed -i "s/ARDUINO_VERSION_PLACEHOLDER/${GITHUB_REF/refs\/tags\//}/" 0008-Imgtool-Append-arduino-to-version-string.patch
```

```suggestion
        run: perl -pi -e "s/ARDUINO_VERSION_PLACEHOLDER/${GITHUB_REF/refs\/tags\//}/g" 0008-Imgtool-Append-arduino-to-version-string.patch
```
Using a consistent placeholder replacement command throughout the workflow will make it easier to understand and maintain.

`sed` and `perl` are confusing enough on their own for dummies like me without having to figure both of them out at the same time!